examples/demo/demo.go

```diff
@@ -389,12 +389,21 @@ func collections(ts *core.Tabs) {
 	ktab, _ := vts.NewTab("Keyed lists")
 	core.NewKeyedList(ktab).SetMap(&mp)
 
-	tbl := make([]*tableStruct, 50)
+	// tbl := make([]*tableStruct, 50)
+	// for i := range tbl {
+	// 	ts := &tableStruct{Age: i, Score: float32(i) / 10}
+	// 	tbl[i] = ts
+	// }
+	// tbl[0].Name = "this is a particularly long field"
+	// ttab, _ := vts.NewTab("Tables")
+	// core.NewTable(ttab).SetSlice(&tbl)
```

Comment:
What is your plan for this commented out code?

Reply:
good catch -- forgot to undo